h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -104,7 +104,14 @@ def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_co
         :param float max_runtime_secs: Maximum allowed runtime in seconds for model training. Use 0 to disable.
         :param bool verbose: Print scoring history to stdout. Defaults to False.
         """
+        self._train(x=x, y=y, training_frame=training_frame, offset_column=offset_column, fold_column=fold_column,
+                    weights_column=weights_column, validation_frame=validation_frame, max_runtime_secs=max_runtime_secs, 
+                    ignored_columns=ignored_columns, model_id=model_id, verbose=verbose)
 
+
+    def _train(self, x=None, y=None, training_frame=None, offset_column=None, fold_column=None,
+              weights_column=None, validation_frame=None, max_runtime_secs=None, ignored_columns=None,
+              model_id=None, verbose=False, extend_parms_fn=None):
```

Comment:
needed to "hide" train implementation to add a hook for subclasses (StackedEnsembleEstimator)